app/templates/serviceLearning/slcManagment.html

```diff
@@ -1,3 +1,4 @@
+{%set title = "Service Learning Courses Management"%}

```

Comment:
I would abbreviate all of the service learning pages to SLC Title. like, "SLC Management"